src/main/java/com/couchbase/lite/DatabaseUpgrade.java

```diff
@@ -83,15 +83,10 @@ protected boolean importData() {
             }
 
             // Open destination database:
-            boolean isOpen = false;
-            Throwable error = null;
             try {
-                isOpen = db.open();
+                db.open();
             } catch (CouchbaseLiteException e) {
-                error = e;
-            }
-            if (!isOpen) {
-                Log.e(TAG, "Upgrade failed:  Couldn't open new db: %s", db.toString(), error);
+                Log.e(TAG, "Upgrade failed:  Couldn't open new db: %s", db.toString(), e);
```

Comment:
I have noticed that DatabaseUpgrade.java has several Log method calls that I'm not sure about the original attention. For example,

```
Log.w(TAG, "Couldn't import local doc '%s': '%s'", docID, e);
```
There are several like the example above. All of these should be revise.